pkg/asset/stock/stock.go

```diff
@@ -5,13 +5,15 @@ import (
 	"os"
 
 	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/kubeconfig"
 	"github.com/openshift/installer/pkg/asset/tls"
 )
 
 // Stock is the stock of assets that can be generated.
 type Stock struct {
 	installConfigStock
 	tlsStock
+	kubeconfigStock
```

Comment:
nit: Is it worth trying to keep these ordered by dependencies (which seems to be what you're aiming for with this placement)?  I expect that's going to get complicated, and we'll probably end up investing less brainpower if we just alphabetize these properties ;).